### PR TITLE
Add direct media support

### PR DESCRIPTION
**Pull request by vadymmarkov**
Update the demo app according to the latest API changes:
- Handle new conference events
- Show splash screen when needed
- Implement call transfer logic

### CODE REVIEW

File: Examples/Conference/SwiftUI/Shared/Screens/Chat/ChatMessageStore.swift [hunk 1]
```diff
@@ -49,7 +49,7 @@ final class ChatMessageStore: ObservableObject {
             return false
         }
 
-        return try await !chat.sendMessage(text)
+        return try await chat.sendMessage(text)
```
Comment (drymarau) [inline Examples/Conference/SwiftUI/Shared/Screens/Chat/ChatMessageStore.swift:52]: What was the reason for flipping the return value here?

**Review by drymarau — APPROVED** [review]
Looks great, had a question you can find below.

File: Examples/Conference/SwiftUI/Shared/Screens/Chat/ChatMessageStore.swift (hunk 1 above)
Comment (vadymmarkov) [reply Examples/Conference/SwiftUI/Shared/Screens/Chat/ChatMessageStore.swift:52]: It was a bug in the chat view, the error icon was shown when the message was successfully sent.

Comment (drymarau) [reply Examples/Conference/SwiftUI/Shared/Screens/Chat/ChatMessageStore.swift:52]: Thanks for clarification!

**Comment by vadymmarkov** [comment]
@drymarau FYI, added one more commit to make use of new `VideoContentMode.fit` https://github.com/pexip/pexip-swift-sdk/pull/12/commits/a34b65c1a73e78f6b33e3033f4ff5797144f2f0e. It makes it possible to show portrait video in correct size.

**Pull request merged**